ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-ref.yaml

```diff
@@ -0,0 +1,12 @@
+ref:
+  as: baremetalds-packet-setup
```

Comment:
Reading through the setup and teardown I might be missing something, but why is it specific to baremetal? It's fine to leave it here for now if it's a big deal, but I think it should really be a step anyone can use to get a packet host

Reply:
Instance type, locations and OS are specific for our case, and currently they are not customizable

Reply:
How about making a generic job that has a case on the cluster_type?

Like https://github.com/andfasano/release/blob/c6e6c1d52b6d7bbd8fcb35fa1c72fbe7211e9830/ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh#L51

I realize it's not perfect, but if we don't make this a top-level step the next job that uses Packet may mimic us and copy everything, and then we end up with fragmentation